services/cd-service/Makefile

```diff
@@ -22,7 +22,7 @@ MAKEFLAGS += --no-builtin-rules
 
 export CGO_ENABLED=1
 
-IMAGENAME?=$(IMAGE_REGISTRY)/kuberpult-cd-service:$(VERSION)
+IMAGENAME?=$(IMAGE_REGISTRY)/kuberpult/kuberpult-cd-service:$(VERSION)
```

Comment:
please change it also in the file `.github/workflows/execution-plan-snippet.yml`